extraConfigDpuInfra.py

```diff
@@ -40,15 +40,21 @@ def install_remotelyh(ip, links):
 
     print("result:", rh.run("sudo rpm-ostree"))
 
+    current_kernel = rh.run("uname -a | awk {'print $3'}").out
     cmd = f"sudo rpm-ostree override replace {wd}/*.rpm"
     print(cmd)
     while True:
-        ret = rh.run(cmd).out.strip().split("\n")
-        if ret and ret[-1] == 'Run "systemctl reboot" to start a reboot':
+        out, err, ret = rh.run(cmd)
+        out = out.strip().split("\n")
+        if out and out[-1] == 'Run "systemctl reboot" to start a reboot':
             break
         else:
-            print(ret)
-            print("Output was something unexpected")
+            print(out)
+            print(err)
+            if f"cannot install both kernel-core-{want} and kernel-core-{current_kernel}" in err:
+                cmd = f"sudo rpm-ostree override replace {wd}/*.rpm --remove=kernel-modules-core-{current_kernel}"
+
```

Comment:
This piece of code is not robust enough. Any `rpm-ostree` might fail because the BF is too slow in some cases. 

Instead of being reactive, first we need to figure out if we are upgrading from 8.6 to 9.2 or the other way around. Depending on that, we need to execute a set of commands. If any of those commands then fail, it's probably because there was a timeout and in that case just rerun them. I'd like to have that behavior in the code too.